static/js/jira-app-creation.js

```diff
@@ -0,0 +1,40 @@
+/* globals $, AP */
```

Comment:
This is a duplicate of `jira-select-card-option.js`. All we need here is the logic for click event on `optionBtn`. 
I think we can simply move that logic inside `jira-select-card-option.js, using the `id` selector of the button instead of the using `.optionBtn`(`class`).

Reply:
I've added a _TODO_ in my PR for this part. Can check it out here:
https://github.com/atlassian/github-for-jira/blob/ef01dae8ae0c1451191dc05f1cd0cb2aabad4fba/static/js/jira-select-card-option.js